pkg/ccl/backupccl/restore.go

```diff
@@ -810,8 +810,8 @@ func splitAndScatter(
 			}
 
 			select {
-			case <-g.Done:
```

Comment:
Why is g.Done removed here but still used in backup.go?

Reply:
Replied in reviewable.